microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -1036,6 +1036,32 @@ See also:
 * [ICoreWebView2_2::WebResourceResponseReceived event (add](/microsoft-edge/webview2/reference/win32/icorewebview2_2#add_webresourceresponsereceived), [remove)](/microsoft-edge/webview2/reference/win32/icorewebview2_2#remove_webresourceresponsereceived)
    * [ICoreWebView2WebResourceResponseReceivedEventArgs interface](/microsoft-edge/webview2/reference/win32/icorewebview2webresourceresponsereceivedeventargs)
 
+The `CustomSchemeRegistration` allows registration of custom schemes in WebView2 so that app can handle the `WebResourceRequested` event for requests to those custom scheme URLs and navigate the WebView2 to it.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2CustomSchemeRegistration Class](/dotnet/api/microsoft.web.webview2.core.corewebview2customschemeregistration?view=webview2-dotnet-1.0.1587.40&preserve-view=true)
+* [CoreWebView2EnvironmentOptions Class](/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions?view=webview2-dotnet-1.0.1587.40&preserve-view=true)
+   * [CoreWebView2EnvironmentOptions.CustomSchemeRegistrations Property](/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions.customschemeregistrations?view=webview2-dotnet-1.0.1587.40&preserve-view=true)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2CustomSchemeRegistration Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2customschemeregistration?view=webview2-winrt-1.0.1587.40&preserve-view=true)
+* [CoreWebView2EnvironmentOptions Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions?view=webview2-winrt-1.0.1587.40&preserve-view=true)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2CustomSchemeRegistration interface](/microsoft-edge/webview2/reference/win32/icorewebview2customschemeregistration?view=webview2-1.0.1587.40&preserve-view=true)
```

Comment:
There is a harmless inconsistency between tabs/platforms re: whether to include the 7 links for the  ICoreWebView2CustomSchemeRegistration members:

We probably could optionally omit the per-member links for Win32, as we did for .NET, since these are small, focused, dedicated types.  In contrast, when we make announcements about members in a giant class that has hundreds of unrelated members, then we must link to specific members.

Should we:
* Keep the 7 member links in this Win32 section, and keep them omitted from .net / winrt tabs?
* Remove these 7 member links from this win32 tab?
* Add the equivalent links in .net & winrt tabs

Reply:
@nishitha-burman - I will enter a Suggested Change to remove the unneeded member links from Win32 tab.  The alternative would be to add equivalent links in .NET & WinRT tabs.  

The design needs to be fairly/reasonably equivalent across the 3 tabs, for maintenance.